docker-compose.yml

```diff
@@ -53,8 +53,8 @@ services:
     container_name: opev-celery
     environment:
       <<: *environment-vars
-      DEPLOYMENT: celery
       C_FORCE_ROOT: "true"
+    command: run celery
```

Comment:
Let's just use celery, instead of run celery

Reply:
And api should start when no argument is passed and also when `api` argument is passed

Reply:
Ok doing the change.